src/components/Section.tsx

```diff
@@ -0,0 +1,39 @@
+import { Box, Typography, useMediaQuery } from '@mui/material';
+
+import theme from 'theme';
+
+type TeamSectionProps = {
+  title: string;
+  subtitle: string;
+  description: string;
+};
+export const Section = (props: TeamSectionProps) => {
+  const isMobile = useMediaQuery(theme.breakpoints.down(750));
+  return (
+    <Box textAlign={'left'} marginRight={isMobile ? '' : '52px'}>
+      <Typography
+        variant="h3"
+        color="text.primary"
+        align="left"
+        sx={{
+          fontWeight: { xs: 600, md: 'bold' },
+          fontSize: { xs: '45px', sm: '45px' },
+        }}
+      >
+        {props.subtitle}
```

Comment:
You can destruct the properties from the props above so you can use "subtitle" etc